app/models/user.rb

```diff
@@ -37,6 +37,14 @@ class User < ActiveRecord::Base
 
   attr_accessor :asserted_attributes
 
+  def personal_key
```

Comment:
We can use `alias_attribute` here (and remove the `personal_key=` as well:

https://apidock.com/rails/Module/alias_attribute